towerlib/towerlib.py

```diff
@@ -110,6 +111,7 @@ def __init__(self,
                  password,
                  secure=False,
                  ssl_verify=True,
+                 timeout=5,
```

Comment:
Let's not break positional compatibility for people and move this to the end of the arguments maybe?

Reply:
Done